hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/CacheConfig.java

```diff
@@ -87,6 +87,15 @@
   public static final String CACHE_COMPACTED_BLOCKS_ON_WRITE_KEY =
       "hbase.rs.cachecompactedblocksonwrite";
 
+  /**
+   * Configuration key to determine total size in bytes of compacted files beyond which we do not cache blocks on compaction
+   */
+  public static final String CACHE_COMPACTED_BLOCKS_ON_WRITE_THRESHOLD_KEY =
+      "hbase.rs.cachecompactedblocksonwritethreshold";
+
+  /**
+   * Configuration key to determine threshold beyond which cache on compaction should not work
```

Comment:
Is this new comment make sense for below existing conf name? Am I missing some thing?

Reply:
Agree, I believe this can be removed